pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -210,8 +210,32 @@ func (o *TriggersOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, arg
 		Flatten()
 
 	output := kcmdutil.GetFlagString(cmd, "output")
-	if len(output) != 0 {
-		o.PrintObject = func(obj runtime.Object) error { return f.PrintObject(cmd, mapper, obj, o.Out) }
+	if len(output) > 0 {
+		o.PrintObject = func(infos []*resource.Info) error {
```

Comment:
I was not sure if there was a better way of approaching this, although this command has a function `printObjects` for cases where no `-o` option is specified, it still did not handle every outputFormat, such as `wide` without this update